rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -834,6 +838,90 @@ void getSvpSpendTxHashUnsigned_whenHashIsCached_shouldReturnTheCachedHash() {
         }
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("save, set and get svp spend transaction waiting for signatures tests")
+    class SvpSpendTxWaitingForSignaturesTests {
+        private final Keccak256 rskTxHash = PegTestUtils.createHash3(1);
+        private final BtcTransaction svpSpendTx = new BtcTransaction(mainnetBtcParams);
+        private final String fieldName = "svpSpendTxWaitingForSignatures";
```

Comment:
no big deal, but i found fieldName not very specific. Maybe `svpSpendTxWaitingForSignaturesField`? A bit redundant but more descriptive. Wdyt?

Reply:
Sounds good, just updated